Tests.Integration.Presentation.Web/ItSystem/ItSystemUsageMigrationTests.cs

```diff
@@ -519,6 +531,100 @@ public async Task PostMigration_To_Own_System_Does_Nothing()
             }
         }
 
+
+        [Fact]
+        public async Task PostMigration_When_System_Has_Relation_With_From_System_Exposing_Interface()
+        {
+            //Arrange
+            var fromItSystem = await CreateSystemAsync();
+            var fromItSystemUsage = await TakeSystemIntoUseAsync(fromItSystem);
+            var toItSystem = await CreateSystemAsync();
+            var toItSystemUsage = await TakeSystemIntoUseAsync(toItSystem);
+            var createdInterface = await CreateInterfaceAsync();
+            var exhibit = await CreateExhibitAsync(createdInterface, toItSystem);
+
+            var migrateToItSystem = await CreateSystemAsync();
+
+            var relation = await CreateSystemRelation(fromItSystemUsage.Id, toItSystemUsage.Id, A<string>(), exhibit.Id, null);
```

Comment:
exhibit.Id er ikke længere sandt. Du skal altid bruge exhibit.interfaceId (eller hvad den nu hedder for tiden). Det andet var "working by chance". Det går jo igen igennem filen, så det skal lige snittes korrekt så man ikke sidder med den forkerte opfattelse ved senere tilretning.